lib/compat/wordpress-6.0/block-template-utils.php

```diff
@@ -26,6 +26,19 @@ function gutenberg_is_theme_directory_ignored( $path ) {
 	return false;
 }
 
+/**
+ * Recursively applies ksort to an array.
+ * From tests/phpunit/tests/theme/wpThemeJsonResolver.php
+ */
+function gutenburg_recursive_ksort( &$array ) {
```

Comment:
gutenburg?

Reply:
Just to namespace it 🤷

Reply:
```suggestion
function gutenberg_recursive_ksort( &$array ) {
```

:)

Reply:
oh whoops i've been doing that all over the place 😬